### ISSUE TITLE
Feature Request: remove/hide the header

### ISSUE THREAD

**Issue by rachelmrgls** [motss/app-datepicker#119]
Hi motss, 

I'm working on a web app using your date picker, and I'd like to use a simpler version, without the header (selected-fulldate) containing the selected year and selected date, so the app-datepicker is just the calendar view. 

![screenshot from 2018-03-20 11-41-10](https://user-images.githubusercontent.com/8468760/37675886-a5e32bc0-2c33-11e8-9314-9cc7c6e4a1c0.png)

I understand this would mean the user can't access the year switcher, but that's ok for my use case, because they can change years by incrementing months. For my use case, users rarely need to input a date more than a year away, and the header takes up a lot of space.

Would this be a feature you'd be willing to add to the datepicker? If so, I'd be happy to implement it.

**Comment by motss** [comment]
Good day, @rachelmrgls!

Thanks for the feedback.

There are 2 ways I can think of to achieve that:

1. **Easy fix** - Make use CSS to set `display: none` on the header
2. **Preferable fix** - To add new property to decide whether or not the header should be rendered.

Hmm...I think the latter is the proper one. This should not be a complicated feature. 😄

**Comment by torendo** [comment]
Right now it can be achieved like this:
```
    let sheet = css`
      .datepicker-header {
        display: none;
      }
    `;
    datepicker.shadowRoot.adoptedStyleSheets = [...datepicker.shadowRoot.adoptedStyleSheets, sheet.styleSheet];
```

**Comment by motss** [comment]
@rachelmrgls @torendo 

I recently encountered such use case in a real project and never thought that it was even a thing. After spending days of pondering on that (and I know it has been almost 2 years), I've decided to introduce inline mode rendering for `<app-datepicker` by adding a new `inline` property in c81640cd668024472940eac0e9eb660ef1918f2d for `v4`.

It might not even get implemented in `v3`. In other words, highly recommend all users to use `v4`.

Thank you for your input and suggestion as workaround before this addition. Really appreciate that. 👍
